source/_integrations/template.markdown

```diff
@@ -201,25 +202,16 @@ select:
   required: true
   type: map
   keys:
-    state:
-      description: Template for the select's current value.
-      required: true
-      type: template
-    select_option:
-      description: Defines actions to run to select an option from the `options` list. The variable `option` will contain the option selected.
+    press:
+      description: Defines actions to run to press the button.
```

Comment:
Did you delete select-specific fields here and add button field instead? Looks like a mistake.

Reply:
🤦🏾‍♂️ thanks for calling that out!